fastlane/metadata/android/en-US/short_description.txt

```diff
@@ -1 +1 @@
-Extract APKs from your device, even if installed from the Playstore. Root access is required for paid apps.
```

Comment:
Short description must be less than 80 chars at least in the Play Store. I haven't found any limits for F-Droid (their website is currently down), but I think they use the same limits.